docs/development/renovate-tutorial.md

```diff
@@ -0,0 +1,145 @@
+# Renovate Hands-on-tutorial
+
+## Introduction
+
+We will learn how to run renovate and experience how it works.
+There are Multiple ways to run Renovate, using CLI, self-hosted server or docker,
+but the easiest way to start with is the [Renovate App](https://github.com/apps/renovate).
+
+You can trigger renovate manually from CLI or docker by running it to pick up any changes needed,
+or you can run [Renovate App](https://github.com/apps/renovate) or a host renovate on a server,
+and it will pick up any needed changes every few minutes/hours.
+
+## Tutorial
+
+- We will work with Github for this tutorial since our app is hosted on Github, open your Github repository
+- Fork [this](https://github.com/PhilipAbed/RenovateTutorial) repository
+- Make sure `issues` are enabled on github, `settings` ->`general` -> check the `issues` checkbox.
+- Install renovate app, go to [Renovate App](https://github.com/apps/renovate), and install it, it will take you to a page where you can select on which repositories you want to run renovate on
+- Select the forked repository, you can configure that at anytime by going again to [Renovate App](https://github.com/apps/renovate) and pressing the `Configure` button.
+- After installing the app, it will run in the background and trigger on the selected repositories and create an on-boarding PR, Which will add a `renovate.json` configuration file.
+- make sure you read the on-boarding PR, then go ahead and merge the on-boarding pull request.
+- if you go to the [dashboard](https://app.renovatebot.com/dashboard), you will see when renovate runs.
+- Once you merge, renovate will trigger again automatically, and you will notice that it scanned the `package.json` and opened some pull requests.
+- renovate by default will try to get the latest version, but you can restrict updates to a certain version using `allowedVersions`
+
+### Allow specific versions
```

Comment:
This and following headings should come under "Post-onboarding configuration"

Reply:
And why is allowing specific versions the first example? It's not all that common and not the most important thing to learn